sdk/eventhub/Azure.Messaging.EventHubs/stress/src/Infrastructure/Publisher.cs

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Producer;
+using Azure.Messaging.EventHubs.Tests;
+using Microsoft.ApplicationInsights;
+
+namespace Azure.Messaging.EventHubs.Stress
+{
+    internal class Publisher
+    {
+        private static readonly ThreadLocal<Random> RandomNumberGenerator = new ThreadLocal<Random>(() => new Random(Interlocked.Increment(ref _randomSeed)), false);
```

Comment:
Are we using this anywhere?   I think this was originally put in before I pulled stuff into the event generator.   We may be able to remove it.